solidity/contracts/examples/SpecAndRunRequester.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.4.23;
+
+import "../Chainlinked.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract SpecAndRunRequester is Chainlinked, Ownable {
+  bytes32 internal requestId;
+  bytes32 public currentPrice;
+
+  event RequestFulfilled(
+    bytes32 indexed requestId,
+    bytes32 indexed price
+  );
+
+  constructor(address _link, address _oracle) Ownable() public {
+    setLinkToken(_link);
+    setOracle(_oracle);
+  }
+
+  function requestEthereumPrice(string _currency) public {
+    string[] memory tasks = new string[](4);
+    tasks[0] = "httpGet";
```

Comment:
It looks like the adapter type is always compared as lower case in [`adapters/adapter.go`](https://github.com/smartcontractkit/chainlink/blob/master/adapters/adapter.go#L43). How do you feel about making L22 & L23 downcase so that it's consistent with L24 & L25?

Reply:
The adapters are always compared in lowercase because in our deserialization we actually force lower case to make comparisons case insensitive:

https://github.com/smartcontractkit/chainlink/blob/master/store/models/job_spec.go#L180-L189

So the case doesn't matter. I thought this was more readable, but don't feel strongly about the casing and can change it if you do.

Reply:
I feel like because it's an example, consistency helps reduce confusion. I'm not really bothered if they're camelcase or downcase. I'd just prefer that they're all the same.

Reply:
I overlooked L24 & 25. Good call, I'll push the changes.